auslib/db.py

```diff
@@ -1465,7 +1465,13 @@ def _versionMatchesRule(self, ruleVersion, queryVersion):
         self.log.debug('ruleVersion: %s, queryVersion: %s', ruleVersion, queryVersion)
         if ruleVersion is None:
             return True
-        return version_compare(queryVersion, ruleVersion)
+        rulesVersionList = ruleVersion.split(",")
+        if 1 == len(rulesVersionList):
+            return version_compare(queryVersion, rulesVersionList[0])
+        versionMatchesRule = False
+        for rule in rulesVersionList:
+            versionMatchesRule = versionMatchesRule or version_compare(queryVersion, rule)
```

Comment:
I don't think there's a need for versionMatchesRule here - simply returning True if version_compare() returns True is fine. If you already found a match, there's no point in testing the remaining values!